dict/P/pandunia.tsv

```diff
@@ -1622,6 +1636,7 @@ PWN:tiger.n.02		bagre	bagr'e	hin:बाघ (bāgh), ben:বাঘ (bagh); ব্
 PWN:tile.n.01		tegul	teg·ul	
 PWN:time.n.01		tem	tem	
 PWN:time.n.01		tempo	temp’o	eng:time, fra:temps, spa:tiempo, por:tempo
+PWN:timeline.n.01		temline	tem·lin'e	

```

Comment:
The word _tem_ means 'an instance or single occasion for some event' (PWN:time.n.01). This word should probably use the sense 'The dimension of the physical universe which, at a given place, orders the sequence of events. The continuum of past-present-future.' (PWN:time.n.05). The word is currently missing in the Pandunia word list but I will find one. See my next commits.